torch/_dynamo/variables/user_defined.py

```diff
@@ -854,6 +854,27 @@ def odict_getitem(self, tx, key):
         )(collections.OrderedDict.__getitem__(self.value, key.as_python_constant()))
 
 
+class DebuggingVariable(UserDefinedObjectVariable):
+    @staticmethod
+    def is_matching_object(obj):
+        import warnings
+
+        return obj in (warnings.warn, print)
+
+    def call_function(self, tx, args, kwargs):
+        if tx.export:
+            # For export cases, we can just make debugging functions no-ops
+            return
+        elif torch._dynamo.config.reorder_prints:
+            tx.debug_locals.append((self, list(args)))
+            return
+        else:
+            return super().call_function(tx, args, kwargs)
```

Comment:
I think this will just call `unimplemented()`.  Let's change the error message to suggest people cal set `config.reorder_prints=True` to fix it.